### PR TITLE
[COM-1753]  Tag: Add new tokens and variants. 

### PR DESCRIPTION
**Pull request by Claratj**
## Please delete checklist items and sections that are not relevant

**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**What is the ticket / issue associated with this change?** ([Link](https://impulsumvc.atlassian.net/browse/COM-1753))

**Other information**:

**Storybook Link** ([Link](https://5eed577398d08d0022b52d76-zsybfpadhu.chromatic.com/))

### CODE REVIEW

File: .changeset/yellow-fireants-try.md
Comment (tkanzakic) [inline .changeset/yellow-fireants-try.md:1]: why do we have two changeset for this pull request?

File: packages/components/src/components/dataDisplay/Tag/Tag.stories.tsx [hunk 1]
```diff
@@ -13,64 +13,64 @@ export default {
...
+              ))}
+            </Tr>
+          ))}
+        </>
       ))}
     </Tbody>
   </Table>
 );
-
-const Template = ({ showIcon, iconPosition, label, ...args }) => (
```
Comment (tkanzakic) [inline packages/components/src/components/dataDisplay/Tag/Tag.stories.tsx:60]: we should get this back

File: packages/components/src/components/dataDisplay/Tag/types.ts [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import { TagProps as ChakraTagProps, ResponsiveValue } from '@chakra-ui/react';
 
 export const tagSizes = ['s', 'm', 'l'] as const;
+export const tagVariants = ['subtle', 'outline', 'solid', 'custom'] as const;
 
 export type TagSize = (typeof tagSizes)[number];
+export type TagVariant = (typeof tagVariants)[number] | ChakraTagProps['variant'];
```
Comment (tkanzakic) [inline packages/components/src/components/dataDisplay/Tag/types.ts:7]: why do we need the `ChakraTagProps` reference in here?

Comment (Claratj) [reply packages/components/src/components/dataDisplay/Tag/types.ts]: not really needed

File: packages/components/src/theme/ComposerTheme/Colors/colors.ts [hunk 3]
```diff
@@ -128,11 +158,4 @@ export const defaultColors = {
   'comms-background-bubble-container-sender': '#FFFFFF',
   'comms-background-bubble-container-receiver': '#F1F1F1',
   'comms-background-bubble-error': '#FFDDD8',
-  'ui-element-outline-hover': '#D7DBE7',
-  'ui-element-outline-focus': '#E6E6E6',
-  'background-action-focus': 'transparent',
-  'text-close-button': '#222222',
-  'text-badge-custom': '#4F4FFF',
-  'text-badge-custom-solid': '#FFFFFF',
-  'background-badge-custom': '#4F4FFF',
```
[the remark this answers is not in the document]
Comment (Claratj) [reply packages/components/src/theme/ComposerTheme/Colors/colors.ts:137]: reordered

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**